pkg/loadbalancer/loadbalancer.go

```diff
@@ -466,6 +466,15 @@ func (b *Backend) String() string {
 	return "[" + b.L3n4Addr.String() + "," + "State:" + state + "]"
 }
 
+// WeightInfo implements interface needed for maglev to compute
+// the maglev lookup tables for backends.
+func (b *Backend) WeightInfo() (id BackendID, addr L3n4Addr, weight uint16) {
```

Comment:
nit: named returns here seems superfluous.

Reply:
FYI: This part of the PR will most likely be moved to a separate PR.

This was authored by @joamaki's, but I think named arguments are useful here for documentation purposes because neither the function name nor the doc-string specify what the function computes. One can probably infer that from the types, although `uint16` is not very descriptive...

Reply:
Moved to https://github.com/cilium/cilium/pull/35885, resolving.